services/configuration.go

```diff
@@ -16,10 +16,12 @@ const (
 	ConfigurationKeyHardforkName     types.ExplorerConfigurationKey = "HardforkName"
 )
 
-/***
+/*
+**
 This is the list of possible configurations that can be changed in the explorer administration
 Per default these values will be taken, overridden by the values from the db, if they exist.
-***/
+**
```

Comment:
Is this change intentional? Not saying this is wrong to do.

Reply:
Yes, this is indeed intentional because otherwise https://github.com/gobitfly/eth2-beaconchain-explorer/actions/runs/6094260445/job/16535457583